vsts/vsts.yaml

```diff
@@ -554,15 +554,15 @@ jobs:
       - task: ComponentGovernanceComponentDetection@0
         displayName: "Component Detection"
       
-      - task: securedevelopmentteam.vss-secure-development-tools.build-task-policheck.PoliCheck@1
+      - task: securedevelopmentteam.vss-secure-development-tools.build-task-policheck.PoliCheck@2
         displayName: "Run PoliCheck"
         inputs:
           targetType: F
           optionsRulesDBPath: 'vsts/PolicheckExclusionsDB.mdb'
           optionsSEV: '1|2|3|4'
           optionsPE: 1
       
-      - task: securedevelopmentteam.vss-secure-development-tools.build-task-antimalware.AntiMalware@3
+      - task: securedevelopmentteam.vss-secure-development-tools.build-task-antimalware.AntiMalware@4
```

Comment:
##[warning]* Please upgrade the version of this build task in your build to major version: 4. For more information, please see: https://aka.ms/gdn-v1-deprecation